pkg/controller/volume/persistentvolume/config/types.go

```diff
@@ -28,6 +28,13 @@ type PersistentVolumeBinderControllerConfiguration struct {
 	PVClaimBinderSyncPeriod metav1.Duration
 	// volumeConfiguration holds configuration for volume related features.
 	VolumeConfiguration VolumeConfiguration
+	// VolumeHostCIDRDenylist is a list of CIDRs that should not be reachable by the
+	// controller from plugins.
+	VolumeHostCIDRDenylist []string
+	// VolumeHostAllowLocalLoopback indicates if local loopback hosts (127.0.0.1, etc)
+	// should be allowed when from plugins. This is a reference so that an unset value
+	// can be set for defaulting (see v1alpha1/defaults.go).
```

Comment:
> VolumeHostAllowLocalLoopback indicates if local loopback hosts (127.0.0.1, etc) should be allowed when from plugins.

Sentence fragment?

> This is a reference so that an unset value can be set for defaulting (see v1alpha1/defaults.go).

I think this bit should be moved to v1alpha1/types.go? this var **isn't** a reference.

Reply:
Arg, thank you.